modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -3,7 +3,7 @@
 import Logger from '@jitsi/logger';
 import $ from 'jquery';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
```

Comment:
Why are we doing this? I think this opts us into concurrent roots, which I don't think we are ready to do just yet. Let's keep this just about Node please.

Reply:
This was in order to fix :
![image](https://github.com/jitsi/jitsi-meet/assets/39557534/1c670a42-e16b-4606-abfc-7fa36f1bdb3b)
I reverted for now though to keep this PR for node only.